### DISCUSSION TITLE
Does Sandboxie control boot sector changes?

### DISCUSSION THREAD

**Discussion by 1matin** [sandboxie-plus/Sandboxie#3539] [Documents and notes]
Sandboxie is used as a tool for isolating programs from the underlying operating system, preventing unwanted changes from happening. However, I’m particularly interested in its effectiveness against unwanted boot sector changes. Does Sandboxie protect against such changes? If so, how does it isolate or prevent modifications to the boot sector, considering its critical role in the system startup process?

**Comment by bastik-1001** [comment]
Sandboxie uses a driver, so that sandboxed apps, by default, are not allowed to access the disk directly (this could be configured) and it denies the app to make those changes. I don't know how this works on a detailed technical level, but since the topic has not been replied to, yet, I wanted to reply to it, even though there are others that could give a better reply.

**Comment by DavidXanatos** [comment]
> Sandboxie is used as a tool for isolating programs from the underlying operating system, preventing unwanted changes from happening. However, I’m particularly interested in its effectiveness against unwanted boot sector changes. Does Sandboxie protect against such changes? If so, how does it isolate or prevent modifications to the boot sector, considering its critical role in the system startup process?

Sandboxie prevents applications under its supervision from obtaining write access to block devices and volumes hence it makes it impossible for these applications to modify the boot sector the MFT or any other raw data on the attached harddrives
